raiden_contracts/contract_manager.py

```diff
@@ -82,6 +82,14 @@ def get_event_abi(self, contract_name: str, event_name: str) -> Dict:
         contract_abi = self.get_contract_abi(contract_name)
         return find_matching_event_abi(contract_abi, event_name)
 
+    def get_constructor_argument_types(self, contract_name: str) -> List:
+        abi = self.get_contract_abi(contract_name=contract_name)
+        return [
+            arg['type'] for arg in list(
```

Comment:
Nitpick: I know you only moved this, but if you want this could be cleaned up into a single list comprehension without the nested `list` and `filter` calls.

Reply:
Good idea.  I'll do this in this PR.